### PR TITLE
[TW-1356] capitalize flows

### PR DESCRIPTION
**Pull request by petewitcher-postman**
Following Marketing's lead, we're capitalizing all instances of "flow" that refer to Postman Flows (the product or the entity). Some docs use the lowercase term in a non-Postman Flows context, for example:
- openapi2.md
- openapi3.md
- collaboration-overview.md
- intro-to-collection-runs.md

### CODE REVIEW

File: src/pages/docs/postman-flows/gs/flows-overview.md [hunk 1]
```diff
@@ -33,19 +33,19 @@ contextual_links:
     url: "https://www.postman.com/case-studies/built/"
 ---
 
-**Postman Flows** is a visual tool for building API-driven applications for the [API-first](https://www.postman.com/api-first/) world. You can use flows to chain requests, handle data, and create real-world workflows in your Postman workspace.
+**Postman Flows** is a visual tool for building API-driven applications for the [API-first](https://www.postman.com/api-first/) world. You can use Flow to chain requests, handle data, and create real-world workflows in your Postman workspace.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/postman-flows/gs/flows-overview.md:36]:
```suggestion
**Postman Flows** is a visual tool for building API-driven applications for the [API-first](https://www.postman.com/api-first/) world. You can use Flows to chain requests, handle data, and create real-world workflows in your Postman workspace.
```

File: src/pages/docs/postman-flows/reference/blocks-list.md [hunk 2]
```diff
@@ -137,7 +137,7 @@ Outputs the text that you enter.
 
 <img alt="Start block" src="https://assets.postman.com/postman-labs-docs/all-blocks/new-ux-start-block.png" width="110px"/>
 
-The first block that runs when a flow is run. When deploying to the cloud [running flows on the cloud](/docs/postman-flows/concepts/automatic-runs/), Can be configured to hold information (either as text or in JSON format) for testing locally.
+The first block that runs when a Flow is run. When deploying to the cloud [running Flows on the cloud](/docs/postman-flows/concepts/automatic-runs/), Can be configured to hold information (either as text or in JSON format) for testing locally.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/postman-flows/reference/blocks-list.md:140]:
```suggestion
The first block that runs when a Flow is run. When deploying to the cloud, [running Flows on the cloud](/docs/postman-flows/concepts/automatic-runs/) can be configured to hold information (either as text or in JSON format) for testing locally.
```

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/administration/team-settings.md [hunk 3]
```diff
@@ -57,8 +57,8 @@ Your team profile includes:
 * A list containing links to your team members' profiles
 * A **Team Description** section
 * A **Highlights** section with pinned elements
-* A **Related collections and flows** section with collections and flows using your team's APIs (suggested by Postman)
-* Links to all your team's public collections, APIs, workspaces, and flows
+* A **Related collections and Flows** section with collections and Flows using your team's APIs (suggested by Postman)
```
Comment (jkonrath-postman) [inline src/pages/docs/administration/team-settings.md:60]: This might not be accurate if the UI hasn't changed to uppercase. (Has it?)

**Review by jkonrath-postman — APPROVED** [review]
One minor thing that is probably OK, but LGTM

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

**Pull request merged**